code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -754,6 +754,27 @@
 	new /obj/item/reagent_containers/pill/bloodregen(src)
 	new /obj/item/reagent_containers/pill/bloodregen(src)
 
+/obj/item/storage/pill_bottle/opifex
+	name = "bottle of mixed supplements"
+	icon_state = "pill_rainbow"
+	desc = "A mixed bottle of pills."
+
+
+/obj/item/storage/pill_bottle/opifex/populate_contents()
+	for(var/i in 1 to 5)
+		new /obj/item/reagent_containers/pill/bloodregen(src)
+	for(var/i in 1 to 5)
+		new /obj/item/reagent_containers/pill/prosurgeon(src)
+	for(var/i in 1 to 5)
+		new /obj/item/reagent_containers/pill/bicaridine(src)
```

Comment:
This is picking a random amount right? I'd much rather these much more commonly used pills where locked to 5 (if not more), Can blow through pills of brute/burn/pain meds like nothing.

Reply:
this is spawning 5 of eatch, not randomizing